wami is finished, pleace add it to tldr (#12003)
tldr-pages/tldr: pages/common/wami.md

> # wami - What am I

Reviewer: Write the meaning of the program name in the description.

```suggestion
# wami
```

> A tool that recommends suitable programs for specific tasks. Open-source, easy to use.
> More information: <https://github.com/evait-security/wami>.

- Search the lake and find expanded results in all categories in descending order:

> `wami -a -S desc -s <search-string>`

Reviewer: Use curly braces to write placeholders.

```suggestion
`wami -a -S desc -s {{search-string}}`
```

> - Search GitHub to find expanded results, displayed in descending order.:

Reviewer: ```suggestion
- Search GitHub to find expanded results, displayed in descending order:
```

> `wami -a -S desc --github <search-string>`

Reviewer: ```suggestion
`wami -a -S desc --github {{search-string}}`
```

> - Search GitHub for topics that match the search string.:

Reviewer: ```suggestion
- Search GitHub for topics that match the search string:
```

> `wami --list-topics <search-string>`

Reviewer: ```suggestion
`wami --list-topics {{search-string}}`
```

# wami

> A tool that recommends suitable programs for specific tasks. Open-source, easy to use.
> More information: <https://github.com/evait-security/wami>.

> - Search the lake and find expanded results in all categories in descending order:

Reviewer: ```suggestion
- Find expanded results in all categories from the lake and [S]ort them in descending order:
```

> `wami -a -S desc -s {{search-string}}`

Reviewer: [The names of placeholders cannot contain hyphens](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#naming).

```suggestion
`wami -a -S desc -s {{search_string}}`
```

> `wami --show-[a]ll --[S]ort desc --[s]earch-all {{search_string}}`

Reviewer 2: @sebastiaanspeck @kbdharun the mnemonics should stay in the commands or in the description?

Reviewer: The descriptions as far I know.

- Search GitHub to find expanded results, displayed in descending order:

`wami -a -S desc --github {{search-string}}`

- Search GitHub for topics that match the search string:

> `wami --list-topics {{search-string}}`

Reviewer: ```suggestion
`wami --list-topics {{search_string}}`
```

> - Search the lake for a tool used in pentests to query for default credentials and [s]ort the results in descending order:

Reviewer 2: ```suggestion
- Search the lake for a tool used in pentests to query for default credentials and [S]ort the results in descending order:
```

> - Search the lake for a tool used in pentests to query for default credentials and order desc to get the first match at the bottom:

Reviewer: IMO, the description is a little bit too long (although it does not exceed the [maximum line length](https://github.com/tldr-pages/tldr/blob/main/.markdownlintrc#L5)). What do you guys think? 

```suggestion
- Search the lake for a tool used in pentests to query for default credentials and [S]ort the results in descending order:
```
